ledger-api/grpc-definitions/com/daml/ledger/api/v2/participant_offset.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+
+option java_outer_classname = "ParticipantOffsetOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Describes a specific point on the participant. This is a participant local value:
+// changes synchronized via a specific domain should have different participant offsets
+// on different participants.
```

Comment:
"should" sounds like the participants should try to avoid clashes, but this is not the case here. How about
```
A participant offset is meaningful only in the context of its participant.
Different participants may associate different offsets to the same change synchronized over a domain, and conversely, the same literal participant offset may refer to different changes on different participants.
```

Reply:
I like it a lot, thanks